_posts/2019-12-13-cockpit-209.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Cockpit 209
+author: mvo
+date: 2019-12-13
+tags: cockpit linux
+slug: cockpit-209
+category: release
+summary: Cockpit with Overview improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 209.
+
+### A new design for the Overview page
+
+The landing page has been completely redesigned.  Information is
+grouped into easier to understand panels, health information is much
+more prominent, the resource graphs have been moved to their own page,
+and the hardware information page should now be easier to find.
+
+![Overview](/images/new-overview.png)
+
+### Session timeouts
+
+Cockpit will automatically log you out of your current session after
+15 minutes of inactivity.
+
+![Automatic logout](/images/session-timeout.png)
+
+You can configure the timeout in `/etc/cockpit/cockpit.conf`.  For
```

Comment:
perhaps mention that the units of this new option are "minutes"? like "you can now configure the timeout, in minutes, ..."